discovery-provider/ddl/migrations/0029_usdc_transactions_metadata.sql

```diff
@@ -0,0 +1,4 @@
+ALTER TABLE
+    usdc_transactions_history
+ADD
+    COLUMN IF NOT EXISTS tx_metadata CHARACTER VARYING;
```

Comment:
maybe worth considering varchar(X) for however large this could be. don't need to change this PR though, just worth thinking on

Reply:
I've always used `VARCHAR(n)` but this was a direct copy from the SQL used to create the `audio_transactions_history` table, and I felt it best to keep them consistent. Who knows, maybe we'll stuff a whole JSON blob in there some day 🤷

Reply:
sg!